site/guides/namespace-provisioning-example.md

```diff
@@ -0,0 +1,441 @@
+# Namespace provisioning example via kpt CLI
```

Comment:
Suggest adding the link to the UI version guide, or simply call this "Namespace Provisioning Example"

Reply:
I am working with Chris to get UI guides published. I will take a second pass on reorganizing these once all guides are in.

Thanks for the suggestion.